### ISSUE TITLE
Can this be installed in a conda environment?

### ISSUE THREAD

**Issue by Jonseed** [ThereforeGames/txt2img2img#8]
I noticed that this uses venv. I'm not too familiar with venv. I installed automatic's repo into a conda environment. Can this script be installed and used in a conda environment? Could I just install the packages that are in the venv directly into my conda environment without venv?

**Comment by ThereforeGames** [comment]
> Could I just install the packages that are in the venv directly into my conda environment without venv?

Yes, I believe that would work. If you run into any issues, you may need to tweak the import statement for rembg in txt2img2img.py, or perhaps use some additional command line arg in the web UI launcher.
